dev-docs/bidders/pubmatic.md

```diff
@@ -226,4 +226,8 @@ Publishers should use the `ortb2` method of setting [First Party Data](https://d
 - `ortb2.user.*`
 
 AdUnit-specific data is supported using `AdUnit.ortb2Imp.ext.*`
+
+### Endpoint Compression
+
+This adapter utilizes gzip compression support built into Prebid.js core. For more information, see [Compression Support for Outgoing Requests](https://docs.prebid.org/dev-docs/bidder-adaptor.html#compression-support-for-outgoing-requests)
```

Comment:
the changes are required in bidder code... no settings are required  so what is in the example page?